test/kubernetes/testutils/assertions/objects.go

```diff
@@ -28,16 +29,16 @@ func (p *Provider) EventuallyObjectsNotExist(ctx context.Context, objects ...cli
 	for _, o := range objects {
 		p.Gomega.Eventually(ctx, func(innerG Gomega) {
 			err := p.clusterContext.Client.Get(ctx, client.ObjectKeyFromObject(o), o)
-			innerG.Expect(apierrors.IsNotFound(err)).To(BeTrue(), "object should not be found in cluster")
+			innerG.Expect(apierrors.IsNotFound(err)).To(BeTrue(), fmt.Sprintf("object %v %s.%s should not be found in cluster", o.GetObjectKind().GroupVersionKind().String(), o.GetName(), o.GetNamespace()))
 		}).
 			WithContext(ctx).
-			WithTimeout(time.Second * 10).
+			WithTimeout(time.Second * 20).
 			WithPolling(time.Millisecond * 200).
 			Should(Succeed())
 	}
 }
 
 func (p *Provider) ExpectNamespaceNotExist(ctx context.Context, ns string) {
 	_, err := p.clusterContext.Clientset.CoreV1().Namespaces().Get(ctx, ns, metav1.GetOptions{})
-	p.Gomega.Expect(apierrors.IsNotFound(err)).To(BeTrue(), "namespace should not be found in cluster")
+	p.Gomega.Expect(apierrors.IsNotFound(err)).To(BeTrue(), fmt.Sprintf("namespace %s should not be found in cluster", ns))
```

Comment:
Gomega actually calls fmt for us: https://github.com/onsi/gomega/blob/master/internal/assertion.go#L86, I think we can just pass in the format string as the first argument, and then any other properties 
```suggestion
	p.Gomega.Expect(apierrors.IsNotFound(err)).To(BeTrue(), "namespace %s should not be found in cluster", ns)
```

Reply:
Was this the case or is there a reason we resolved out this suggestion?